src/main/java/com/mraof/minestuck/alchemy/CardCaptchas.java

```diff
@@ -49,7 +48,16 @@ public static String getCaptcha(Item item, MinecraftServer server)
 		MSExtraData data = MSExtraData.get(server);
 		CardCaptchas captchas = data.getCardCaptchas();
 		
-		if(captchas.captchasMap.containsKey(item))
+		if(PredeterminedCardCaptchas.predefinedCardMap.containsKey(item) && captchas.captchasMap.containsKey(item))
+		{
+			captchas.captchasMap.remove(item);
+		}
+		
+		if(PredeterminedCardCaptchas.predefinedCardMap.containsKey(item))
+		{
+			return PredeterminedCardCaptchas.predefinedCardMap.get(item);
+		}
+		else if(captchas.captchasMap.containsKey(item))
 			return captchas.captchasMap.get(item);
```

Comment:
When reaching this point, I think it'd be nice to also check for overlap of the captcha that was found.